test/integration/default/inspec/yum_osuosl_spec.rb

```diff
@@ -0,0 +1,4 @@
+describe yum.repo('osuosl') do
+  it { should exist }
+  it { should be_enabled }
+end
```

Comment:
Can you please just rename this to ``default_spec.rb``?

Reply:
Done